README.md

```diff
@@ -64,37 +64,33 @@ func main() {
 		os.Exit(-1)
 	}
 
-	// 3) Since our FilterByDeviceName Function requires the list of Device Names we would
-	// like to search for, we'll go ahead and define that now.
-	deviceNames := []string{"Random-Float-Device"}
+	// 3) Shows how to access the application's specific configuration settings.
+	deviceNames, err := edgexSdk.GetAppSettingStrings("DeviceNames")
+	if err != nil {
+		edgexSdk.LoggingClient.Error("Unable to retrieve DeviceNames: " + err.Error())
```

Comment:
I dont think you need the log prefix here, just log the error returned since that states the property that was not found.

Reply:
Agreed.